examples/asr/conf/speech_multitask/fast-conformer_aed.yaml

```diff
@@ -132,45 +135,70 @@ model:
     dropout_emb: 0.0 # The dropout used for embeddings
     dropout_att: 0.1 # The dropout for multi-headed attention modules
 
+  # Optional Transformer Encoder sandwitched between ASR Encoder and Transformer Ddcoder.
+  # Only used if num_layers > 0
   transf_encoder:
+    _target_: nemo.collections.asr.modules.transformer.transformer_encoders.TransformerEncoder
     num_layers: 0
     hidden_size: 512
     inner_size: 2048
     num_attention_heads: 8
     ffn_dropout: 0.1
     attn_score_dropout: 0.1
     attn_layer_dropout: 0.1
+    mask_future: False
+    pre_ln: True
+    pre_ln_final_layer_norm: True
 
   transf_decoder:
-    library: nemo
+    _target_: nemo.collections.asr.modules.transformer.get_nemo_transformer
     model_name: null
     pretrained: false
-    max_sequence_length: 512
-    num_token_types: 0
-    embedding_dropout: 0.1
-    learn_positional_encodings: false
-    hidden_size: 1024
-    inner_size: 4096
-    num_layers: 24
-    num_attention_heads: 8
-    ffn_dropout: 0.1
-    attn_score_dropout: 0.1
-    attn_layer_dropout: 0.1
-    hidden_act: relu
-    pre_ln: true
+    encoder: null
     pre_ln_final_layer_norm: true
 
+    config_dict:
```

Comment:
we are using `config_dict` for `transf_decoder` but not for `transf_encoder` (i see the reason why it is way in model init.. so just a note )

Reply:
Yeah the get_transformers function is really sad but its not worth refactoring the function. This is fine for the time being i think..